Application/EdFi.Ods.Api.IntegrationTestHarness/appsettings.json

```diff
@@ -0,0 +1,60 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
+      "Microsoft": "Warning"
+    }
+  },
+  "AllowedHosts": "*",
+  "ConnectionStrings": {
+    "EdFi_Ods": "Server=(local); Database=EdFi_Ods_Populated_Template_Test; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_Admin": "Server=(local); Database=EdFi_Admin_Test; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_Security": "Server=(local); Database=EdFi_Security_Test; Trusted_Connection=True; Persist Security Info=True; Application Name=EdFi.Ods.WebApi;",
+    "EdFi_master": "Server=(local); Database=master; Trusted_Connection=True; Application Name=EdFi.Ods.WebApi;"
+  },
```

Comment:
This appsettings file is wrong, look at the NetCore application and see how it is broken down. ApiSettings is a dto that needs to be populated, and that is from a separate key.

Reply:
Done